ibis/backends/dask/execution/util.py

```diff
@@ -112,6 +114,7 @@ def coerce_to_output(
         ir.DestructScalar,
         ir.StructScalar,
     )
+
```

Comment:
Nit: whitespace

Reply:
This is all formatted with black

Reply:
I am not sure if black will catch extra line here. Usually I'd avoid touch functions that is unrelated to the core change of the PR.